src/pybind/mgr/dashboard/services/rgw_client.py

```diff
@@ -223,6 +223,7 @@ def _get_user_keys(user: str, realm: Optional[str] = None) -> Tuple[str, str]:
                 'user', 'create',
                 '--uid', user,
                 '--display-name', 'Ceph Dashboard',
+                '--caps', 'keys=*'
```

Comment:
@alimaredia oh, this is already a `--system` user. i think our rest apis should treat system users as if they have all caps